ts/doc.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2014 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Matt Tracy (matt.r.tracy@gmail.com)
+
+/*
+Package ts provides a high-level time series API on top of an underlying
+cockroach datastore. Cockroach itself provides a single monolithic, sorted
+key-value map. Package ts collects time series data, consisting of datapoints
+with timestamps, and translates it into the key-value store.
+
+The ts package is a work in progress, and will initially only service queries
```

Comment:
I'd add a `TODO(mtracy)` to this sentence to make it easily grep'able, otherwise the comment is likely to live past its expiration date.